pyrobosim/pyrobosim/core/room.py

```diff
@@ -71,6 +74,29 @@ def __init__(
         else:
             self.nav_poses = [Pose.from_list(self.centroid)]
 
+    def _parse_color(self, color):
```

Comment:
Since this will end up affecting things other than rooms, like hallways/locations/objects/robots, would consider moving this to more of the general utilities.

Reply:
Done.